src/plugins/expressions/common/util/get_by_alias.ts

```diff
@@ -11,16 +11,25 @@
  * the given object/array for a case-insensitive match, which could be either the
  * `name` itself, or something under the `aliases` property.
  */
-export function getByAlias<T extends { name?: string; aliases?: string[] }>(
+
+export const ALL_NAMESPACES = '*';
+
+export function getByAlias<T extends { name?: string; aliases?: string[]; namespace?: string }>(
   node: T[] | Record<string, T>,
-  nodeName: string
+  nodeName: string,
+  nameSpace?: string
```

Comment:
In my experience, using different letters case to differentiate between closures may produce some bugs in the future. And it's super simple to slip that while reading the code. I guess it's better to rename that to `namespace` for consistency and pretty documentation and rename the one below to something like `nodeNamespace`.